docs/en/reference/working-with-associations.rst

```diff
@@ -32,62 +32,66 @@ information about its type and if it's the owning or inverse side.
 .. code-block:: php
 
     <?php
-    /** @Entity */
+    #[Entity]
     class User
     {
-        /** @Id @GeneratedValue @Column(type="string") */
-        private $id;
+        #[Id]
+        #[GeneratedValue]
+        #[Column(type: 'string')]
```

Comment:
Should we recommend one attribute per line, or `#[Id, GeneratedValue, Column(type: 'string')`?

Reply:
Both are fine for me, but in this case I'd lean towards the one-liner.